src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
     public String getRandomColor() {
-        return null;
+        return Color.values()[new Random().nextInt(Color.values().length)].name();
```

Comment:
```suggestion
        return Color.values()[random.nextInt(Color.values().length)].name();
```

Reply:
Make it like this 
![image](https://github.com/user-attachments/assets/c3014aaa-cc52-428e-8db8-6930f5b18de1)